src/mlpack/methods/linear_svm/linear_svm_impl.hpp

```diff
@@ -48,10 +48,35 @@ LinearSVM<MatType>::LinearSVM(
     delta(delta),
     fitIntercept(fitIntercept)
 {
+  if (numClasses == 1)
+  {
+    std::ostringstream oss;
+    oss << "LinearSVM dataset has 0 number of classes!";
```

Comment:
No need for an `ostringstream` here since the error message is just a plain string.

Reply:
Updated Please take a look. :)